scene/gui/tree.cpp

```diff
@@ -4315,6 +4305,16 @@ bool Tree::edit_selected(bool p_force_edit) {
 	return false;
 }
 
+Rect2 Tree::_get_item_focus_rect(const TreeItem *p_item) const {
+	Rect2 rect;
+	if (select_mode == SELECT_ROW) {
+		rect = p_item->get_meta("__focus_col_" + itos(selected_col));
```

Comment:
This should move from meta to some internal property tbh. It's super inefficient.